partner_helper/partner.py

```diff
@@ -39,19 +23,19 @@ def split_char(char, output_number, size):
     return result
 
 
-class ResPartner(orm.Model):
+class ResPartner(models.Model):
     _inherit = "res.partner"
 
     def _get_split_address(
-            self, cr, uid, partner, output_number, max_size, context=None):
+            self, partner, output_number, max_size):
```

Comment:
I think with the new API we should refactor the way to pass the args.
It will be better to have something like that

```
def _get_split_address(self, output_number, max_size):
    self.ensure_one()
    ....
```

So using it will be
```
street1, street2, street3 = picking.partner_id._get_split_address(3, 35)
```